index.html

```diff
@@ -2401,9 +2405,8 @@ <h4>API Specification (Thing Model)</h1>
                     given in this Thing Model
                     are not normative and are provided for information only.
                     </p>
-                    <pre class="advisement json">
-                        <div data-include='directory.tm.json'></div>
-                    </pre>
+                    <pre class="advisement json"
+                         data-include='directory.tm.json'></pre>
```

Comment:
This is change is to remove an extra line-break from the rendered output.

Reply:
Note that I in fact inlined this in the 3-wd publication.